src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.spec.js

```diff
@@ -242,13 +242,13 @@ describe('Reactions.vue', () => {
 					NcPopover,
 				},
 			})
-			reactionsStored['🎄'].push({ actorDisplayName: 'user3', actorId: 'admin', actorType: 'users' })
-			let response = generateOCSResponse({ payload: reactionsStored })
-			addReactionToMessage.mockResolvedValue(response)
+			const addedReaction = [...reactionsStored['🎄'], { actorDisplayName: 'user3', actorId: 'admin', actorType: 'users' }]
```

Comment:
```suggestion
			const addedReaction = {
				...reactionsStored,
				'🎄': [...reactionsStored['🎄'], { actorDisplayName: 'user3', actorId: 'admin', actorType: 'users' }],
			}
```

Otherwise you're passing incorrect payload